TestControllers/TestController.psm1

```diff
@@ -788,6 +788,10 @@ Class TestController
 					-ARMImageName $CurrentTestData.SetupConfig.ARMImageName -OsVHD $global:BaseOsVHD -BuildURL $env:BUILD_URL -TableName $dataTableName
 
 				Upload-TestResultToDatabase -SQLQuery $SQLQuery
+
+				if ($dataTableName -eq "LISAv2Results") {
```

Comment:
why we don't insert results to kusto when table is not "LISAv2Results"?

Reply:
Agreed with Lizzha.

Reply:
It only need to log the general telemetry test results, so we just take the necessary changes in this PR.  If enable other data tables,  need to consider more and complex scenarios in current LISAv2 code.